### PR TITLE
Add flag to decrypt the configuration

### PR DESCRIPTION
**Pull request by rubiojr**
Needs https://github.com/muesli/beehive/pull/295 to be merged first.

### CODE REVIEW

File: watchdog_linux.go [hunk 1]
```diff
@@ -22,14 +22,14 @@ func init() {
 	// returns the configured WatchdogSec in the service unit as time.Duration
 	interval, err := daemon.SdWatchdogEnabled(false)
 	if err != nil || interval == 0 {
-		log.Printf("Systemd watchdog not enabled")
+		log.Debug("Systemd watchdog not enabled")
```
Comment (rubiojr) [inline watchdog_linux.go:25]: Most users won't need to see this an it get in the way when writing the json config to stdout with `--decrypt`.

**Comment by muesli** [comment]
Oh, careful this was a pull request on the aes-config-backend branch. You probably wanted to merge this into master?

File: watchdog_linux.go [hunk 2]
```diff
@@ -22,14 +22,14 @@ func init() {
...
 		return
 	}
 
 	// We want to notify the watchdog every WatchdogSec/3, that is, if WatchdogSec is
 	// set to 30 seconds, we'll send a notification to systemd every 10 seconds.
 	runEvery := interval / 3
-	log.Printf("Systemd watchdog notifications every %.2f seconds", runEvery.Seconds())
+	log.Debug("Systemd watchdog notifications every %.2f seconds", runEvery.Seconds())
```
Comment (muesli) [inline watchdog_linux.go:32]: We probably need to use a `Debugf` here, as this contains a formatting directive.

**Comment by rubiojr** [comment]
> Oh, careful this was a pull request on the aes-config-backend branch. You probably wanted to merge this into master?

Yep, sorry for the confusion. The intention was to have a clear diff then change base branch before merging. Fixing in a sec.

**Comment by rubiojr** [comment]
Superseded by #307

**Pull request merged**